test/Microsoft.AspNetCore.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -289,7 +289,7 @@ public void HtmlAttributes_IgnoresCase(string originalName, string updatedName)
         {
             // Arrange
             var executionContext = new TagHelperExecutionContext("p", TagMode.StartTagAndEndTag);
-            executionContext.HTMLAttributes[originalName] = "hello";
+            executionContext.AddHtmlAttribute(originalName, "hello");
```

Comment:
__Wait__ `Add(string name, object value)` exists. @NTaylorMullen please use that as it's the more obvious replacement for the implicit `string` -> `TagHelperAttribute` conversion.